src/d3a/d3a_core/redis_connections/redis_communication.py

```diff
@@ -41,20 +43,26 @@
 ZIP_RESULTS_CHANNEL = "d3a-zip-results"
 ZIP_RESULTS_KEY = "d3a-zip-results-key/"
 
+if TYPE_CHECKING:
+    from d3a.models.area import Area
+
 
 class RedisSimulationCommunication:
-    def __init__(self, simulation, simulation_id, live_events):
-        if simulation_id is None:
-            return
+    def __init__(self, simulation, live_events):
         self._live_events = live_events
-        self._simulation_id = simulation_id
```

Comment:
I suggest keeping this line `self._simulation_id = d3a.constants.CONFIGURATION_ID`
So you could revert the rest of the lines